V0_0_5__Alter_Asset_Internalize_Properties does not migrate properties into the expected JSON structure

## Describe the bug

I suspect the migration V0_0_5__Alter_Asset_Internalize_Properties does not migrate properties into the expected JSON structure in the new "properties" column in the edc_asset table.

### To Reproduce

1. Take a populated database from a tractus-x edc 0.5.3 as starting point.
2. Start an edc compiled from the current main branch on it.
3. It runs the migrations and starts succesfully.
4. Request an existing asset via the managament API (e.g. GET http://127.0.0.1:9081/management/v3/assets/1_test)

### Expected behavior

I expect to reveice the requested asset in the http response.

### Screenshots/Error Messages

```
SEVERE 2024-01-25T12:56:43.1632415 JerseyExtension: Unexpected exception caught
org.eclipse.edc.spi.persistence.EdcPersistenceException: java.lang.ClassCastException: class java.util.ArrayList cannot be cast to class java.util.Map (java.util.ArrayList and java.util.Map are in module java.base of loader 'bootstrap')
	at org.eclipse.edc.sql.SqlQueryExecutor$1.tryAdvance(SqlQueryExecutor.java:131)
	at java.base/java.util.stream.ReferencePipeline.forEachWithCancel(ReferencePipeline.java:129)
	at java.base/java.util.stream.AbstractPipeline.copyIntoWithCancel(AbstractPipeline.java:527)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:513)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.FindOps$FindOp.evaluateSequential(FindOps.java:150)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.findFirst(ReferencePipeline.java:647)
	at org.eclipse.edc.connector.store.sql.assetindex.SqlAssetIndex.findById(SqlAssetIndex.java:77)
	at org.eclipse.edc.connector.service.asset.AssetServiceImpl.lambda$findById$0(AssetServiceImpl.java:57)
	at org.eclipse.edc.transaction.local.LocalTransactionContext.execute(LocalTransactionContext.java:74)
	at org.eclipse.edc.connector.service.asset.AssetServiceImpl.findById(AssetServiceImpl.java:57)
	at org.eclipse.edc.connector.api.management.asset.v3.AssetApiController.lambda$getAsset$5(AssetApiController.java:110)
	at java.base/java.util.Optional.map(Optional.java:260)
	at org.eclipse.edc.connector.api.management.asset.v3.AssetApiController.getAsset(AssetApiController.java:110)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.glassfish.jersey.server.model.internal.ResourceMethodInvocationHandlerFactory.lambda$static$0(ResourceMethodInvocationHandlerFactory.java:52)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher$1.run(AbstractJavaResourceMethodDispatcher.java:146)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.invoke(AbstractJavaResourceMethodDispatcher.java:189)
	at org.glassfish.jersey.server.model.internal.JavaResourceMethodDispatcherProvider$TypeOutInvoker.doDispatch(JavaResourceMethodDispatcherProvider.java:219)
	at org.glassfish.jersey.server.model.internal.AbstractJavaResourceMethodDispatcher.dispatch(AbstractJavaResourceMethodDispatcher.java:93)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.invoke(ResourceMethodInvoker.java:478)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:400)
	at org.glassfish.jersey.server.model.ResourceMethodInvoker.apply(ResourceMethodInvoker.java:81)
	at org.glassfish.jersey.server.ServerRuntime$1.run(ServerRuntime.java:261)
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:248)
	at org.glassfish.jersey.internal.Errors$1.call(Errors.java:244)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:292)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:274)
	at org.glassfish.jersey.internal.Errors.process(Errors.java:244)
	at org.glassfish.jersey.process.internal.RequestScope.runInScope(RequestScope.java:265)
	at org.glassfish.jersey.server.ServerRuntime.process(ServerRuntime.java:240)
	at org.glassfish.jersey.server.ApplicationHandler.handle(ApplicationHandler.java:697)
	at org.glassfish.jersey.servlet.WebComponent.serviceImpl(WebComponent.java:394)
	at org.glassfish.jersey.servlet.WebComponent.service(WebComponent.java:346)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:357)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:311)
	at org.glassfish.jersey.servlet.ServletContainer.service(ServletContainer.java:205)
	at org.eclipse.jetty.servlet.ServletHolder.handle(ServletHolder.java:764)
	at org.eclipse.jetty.servlet.ServletHandler.doHandle(ServletHandler.java:529)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextHandle(ScopedHandler.java:221)
	at org.eclipse.jetty.server.handler.ContextHandler.doHandle(ContextHandler.java:1381)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:176)
	at org.eclipse.jetty.servlet.ServletHandler.doScope(ServletHandler.java:484)
	at org.eclipse.jetty.server.handler.ScopedHandler.nextScope(ScopedHandler.java:174)
	at org.eclipse.jetty.server.handler.ContextHandler.doScope(ContextHandler.java:1303)
	at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:129)
	at org.eclipse.jetty.server.handler.ContextHandlerCollection.handle(ContextHandlerCollection.java:192)
	at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:122)
	at org.eclipse.jetty.server.Server.handle(Server.java:563)
	at org.eclipse.jetty.server.HttpChannel$RequestDispatchable.dispatch(HttpChannel.java:1598)
	at org.eclipse.jetty.server.HttpChannel.dispatch(HttpChannel.java:753)
	at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:501)
	at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:287)
	at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:314)
	at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:100)
	at org.eclipse.jetty.io.SelectableChannelEndPoint$1.run(SelectableChannelEndPoint.java:53)
	at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:969)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.doRunJob(QueuedThreadPool.java:1194)
	at org.eclipse.jetty.util.thread.QueuedThreadPool$Runner.run(QueuedThreadPool.java:1149)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.ClassCastException: class java.util.ArrayList cannot be cast to class java.util.Map (java.util.ArrayList and java.util.Map are in module java.base of loader 'bootstrap')
	at org.eclipse.edc.connector.store.sql.assetindex.SqlAssetIndex.mapAsset(SqlAssetIndex.java:190)
	at org.eclipse.edc.sql.SqlQueryExecutor$1.tryAdvance(SqlQueryExecutor.java:128)
	... 63 more
```

## Context Information

_Add any other context or information about the problem here._

- Used version: 1f4dc8fed5bf38708d52659a7ff8b77e3ffd8943 (main brach from 2024-01-25)
- OS: Windows
- `java --version`: openjdk 17.0.10 2024-01-16 LTS OpenJDK Runtime Environment Corretto-17.0.10.7.1 (build 17.0.10+7-LTS) OpenJDK 64-Bit Server VM Corretto-17.0.10.7.1 (build 17.0.10+7-LTS, mixed mode, sharing)

The "properties" column of the asset in the database:
`[{"property_name" : "https://w3id.org/edc/v0.0.1/ns/description", "property_value" : "Product EDC Demo Asset", "property_type" : "java.lang.String"}, {"property_name" : "https://w3id.org/edc/v0.0.1/ns/id", "property_value" : "1_test", "property_type" : "java.lang.String"}]`

## Possible Implementation

--- a/edc-extensions/postgresql-migration/build.gradle.kts
+++ b/edc-extensions/postgresql-migration/build.gradle.kts
@@ -30,5 +30,7 @@ dependencies {
     implementation(libs.edc.sql.core)
     runtimeOnly(libs.postgres)
 
-    implementation("org.flywaydb:flyway-core:9.22.3")
+    implementation(libs.flyway.core)
+
+    testImplementation(testFixtures(libs.edc.sql.core))
 }

--- a/edc-extensions/postgresql-migration/src/main/java/org/eclipse/tractusx/edc/postgresql/migration/AbstractPostgresqlMigrationExtension.java
+++ b/edc-extensions/postgresql-migration/src/main/java/org/eclipse/tractusx/edc/postgresql/migration/AbstractPostgresqlMigrationExtension.java
@@ -26,17 +26,15 @@ import org.eclipse.edc.spi.system.ServiceExtension;
 import org.eclipse.edc.spi.system.ServiceExtensionContext;
 import org.eclipse.edc.sql.DriverManagerConnectionFactory;
 import org.eclipse.edc.sql.datasource.ConnectionFactoryDataSource;
-import org.flywaydb.core.Flyway;
-import org.flywaydb.core.api.MigrationVersion;
 
 import java.util.Objects;
 import java.util.Properties;
 
+import static org.flywaydb.core.api.MigrationVersion.LATEST;
+
 abstract class AbstractPostgresqlMigrationExtension implements ServiceExtension {
 
     private static final String EDC_DATASOURCE_PREFIX = "edc.datasource";
-    private static final String MIGRATION_LOCATION_BASE =
-            String.format("classpath:%s", AbstractPostgresqlMigrationExtension.class.getPackageName().replace(".", "/"));
 
     private static final String DEFAULT_MIGRATION_ENABLED_TEMPLATE = "true";
     @Setting(value = "Enable/disables subsystem schema migration", defaultValue = DEFAULT_MIGRATION_ENABLED_TEMPLATE, type = "boolean")
@@ -79,19 +77,8 @@ abstract class AbstractPostgresqlMigrationExtension implements ServiceExtension 
         var driverManagerConnectionFactory = new DriverManagerConnectionFactory();
         var dataSource = new ConnectionFactoryDataSource(driverManagerConnectionFactory, jdbcUrl, jdbcProperties);
 
-        var flyway =
-                Flyway.configure()
-                        .baselineVersion(MigrationVersion.fromVersion("0.0.0"))
-                        .failOnMissingLocations(true)
-                        .dataSource(dataSource)
-                        .table("flyway_schema_history_%s".formatted(subSystemName))
-                        .locations("%s/%s".formatted(MIGRATION_LOCATION_BASE, subSystemName))
-                        .defaultSchema(config.getString(MIGRATION_SCHEMA, DEFAULT_MIGRATION_SCHEMA))
-                        .load();
-
-        flyway.baseline();
-
-        var migrateResult = flyway.migrate();
+        var defaultSchema = config.getString(MIGRATION_SCHEMA, DEFAULT_MIGRATION_SCHEMA);
+        var migrateResult = FlywayManager.migrate(dataSource, subSystemName, defaultSchema, LATEST);
 
         if (!migrateResult.success) {
             throw new EdcPersistenceException(

--- a/edc-extensions/postgresql-migration/src/main/java/org/eclipse/tractusx/edc/postgresql/migration/FlywayManager.java
+++ b/edc-extensions/postgresql-migration/src/main/java/org/eclipse/tractusx/edc/postgresql/migration/FlywayManager.java
@@ -1,0 +1,45 @@
+/*
+ *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.tractusx.edc.postgresql.migration;
+
+import org.flywaydb.core.Flyway;
+import org.flywaydb.core.api.MigrationVersion;
+import org.flywaydb.core.api.output.MigrateResult;
+
+import javax.sql.DataSource;
+
+
+public class FlywayManager {
+
+    private static final String MIGRATION_LOCATION_BASE =
+            String.format("classpath:%s", FlywayManager.class.getPackageName().replace(".", "/"));
+
+    public static MigrateResult migrate(DataSource dataSource, String subSystemName, String defaultSchema, MigrationVersion target) {
+        var flyway =
+                Flyway.configure()
+                        .baselineVersion(MigrationVersion.fromVersion("0.0.0"))
+                        .failOnMissingLocations(true)
+                        .dataSource(dataSource)
+                        .table("flyway_schema_history_%s".formatted(subSystemName))
+                        .locations("%s/%s".formatted(MIGRATION_LOCATION_BASE, subSystemName))
+                        .defaultSchema(defaultSchema)
+                        .target(target)
+                        .load();
+
+        flyway.baseline();
+
+        return flyway.migrate();
+    }
+}

--- a/edc-extensions/postgresql-migration/src/main/resources/org/eclipse/tractusx/edc/postgresql/migration/asset/V0_0_6__Transform_Property_Array_To_Map.sql
+++ b/edc-extensions/postgresql-migration/src/main/resources/org/eclipse/tractusx/edc/postgresql/migration/asset/V0_0_6__Transform_Property_Array_To_Map.sql
@@ -1,0 +1,16 @@
+--
+--  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft
+--
+--  This program and the accompanying materials are made available under the
+--  terms of the Apache License, Version 2.0 which is available at
+--  https://www.apache.org/licenses/LICENSE-2.0
+--
+--  SPDX-License-Identifier: Apache-2.0
+--
+--  Contributors:
+--       Bayerische Motoren Werke Aktiengesellschaft - Migration into one table
+--
+
+UPDATE edc_asset SET
+properties = (select json_object_agg(json_array_elements->>'property_name', json_array_elements->>'property_value') from json_array_elements(properties) as json_array_elements),
+private_properties = (select json_object_agg(json_array_elements->>'property_name', json_array_elements->>'property_value') from json_array_elements(private_properties) as json_array_elements)

--- a/edc-extensions/postgresql-migration/src/test/java/org/eclipse/tractusx/edc/postgresql/migration/AssetPostgresqlMigrationExtensionTest.java
+++ b/edc-extensions/postgresql-migration/src/test/java/org/eclipse/tractusx/edc/postgresql/migration/AssetPostgresqlMigrationExtensionTest.java
@@ -1,0 +1,78 @@
+/*
+ *  Copyright (c) 2024 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.tractusx.edc.postgresql.migration;
+
+import org.eclipse.edc.connector.store.sql.assetindex.SqlAssetIndex;
+import org.eclipse.edc.connector.store.sql.assetindex.schema.postgres.PostgresDialectStatements;
+import org.eclipse.edc.junit.annotations.PostgresqlDbIntegrationTest;
+import org.eclipse.edc.spi.types.TypeManager;
+import org.eclipse.edc.spi.types.domain.asset.Asset;
+import org.eclipse.edc.sql.QueryExecutor;
+import org.eclipse.edc.sql.testfixtures.PostgresqlStoreSetupExtension;
+import org.flywaydb.core.api.MigrationVersion;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import java.util.Map;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@PostgresqlDbIntegrationTest
+@ExtendWith(PostgresqlStoreSetupExtension.class)
+class AssetPostgresqlMigrationExtensionTest {
+
+    private SqlAssetIndex store;
+
+    @BeforeEach
+    void setUp(PostgresqlStoreSetupExtension extension, QueryExecutor queryExecutor) {
+        store = new SqlAssetIndex(extension.getDataSourceRegistry(), extension.getDatasourceName(),
+                extension.getTransactionContext(), new TypeManager().getMapper(), new PostgresDialectStatements(),
+                queryExecutor);
+    }
+
+    @Test // bugfix https://github.com/eclipse-tractusx/tractusx-edc/issues/1003
+    void version006shouldTransformPropertiesListToMap(PostgresqlStoreSetupExtension extension) {
+        var dataSource = extension.getDataSourceRegistry().resolve(extension.getDatasourceName());
+        FlywayManager.migrate(dataSource, "asset", "public", MigrationVersion.fromVersion("0.0.5"));
+
+        insertAsset(extension, "1");
+        insertAsset(extension, "2");
+
+        FlywayManager.migrate(dataSource, "asset", "public", MigrationVersion.fromVersion("0.0.6"));
+
+        var result = store.findById("1");
+
+        assertThat(result).isNotNull();
+        assertThat(result.getProperties()).containsExactlyInAnyOrderEntriesOf(
+                Map.of(Asset.PROPERTY_ID, "1", "key", "value1", "anotherKey", "anotherValue1"));
+        assertThat(result.getPrivateProperties()).containsExactlyInAnyOrderEntriesOf(
+                Map.of("privateKey", "privateValue1", "anotherPrivateKey", "anotherPrivateValue1"));
+    }
+
+    private void insertAsset(PostgresqlStoreSetupExtension extension, String id) {
+        var propertiesArray = "[ %s, %s ]".formatted(propertyJsonMap("key", "value" + id), propertyJsonMap("anotherKey", "anotherValue" + id));
+        var privatePropertiesArray = "[ %s, %s ]".formatted(propertyJsonMap("privateKey", "privateValue" + id), propertyJsonMap("anotherPrivateKey", "anotherPrivateValue" + id));
+
+        extension.runQuery(("insert into edc_asset (asset_id, properties, private_properties, data_address) " +
+                "values ('%s', '%s'::json, '%s'::json, '{\"type\":\"type\"}'::json)")
+                .formatted(id, propertiesArray, privatePropertiesArray));
+    }
+
+    private String propertyJsonMap(String key, String value) {
+        return "{\"property_name\" : \"%s\", \"property_value\" : \"%s\", \"property_type\" : \"java.lang.String\"}".formatted(key, value);
+    }
+
+}

--- a/edc-tests/e2e-tests/src/test/java/org/eclipse/tractusx/edc/tests/edr/AbstractDeleteEdrTest.java
+++ b/edc-tests/e2e-tests/src/test/java/org/eclipse/tractusx/edc/tests/edr/AbstractDeleteEdrTest.java
@@ -15,12 +15,8 @@
 package org.eclipse.tractusx.edc.tests.edr;
 
 import jakarta.json.Json;
-import okhttp3.mockwebserver.MockWebServer;
-import org.assertj.core.api.Condition;
 import org.eclipse.edc.policy.model.Operator;
 import org.eclipse.tractusx.edc.lifecycle.tx.TxParticipant;
-import org.junit.jupiter.api.AfterEach;
-import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
@@ -50,12 +46,6 @@ public abstract class AbstractDeleteEdrTest {
             .name(PLATO_NAME)
             .id(PLATO_BPN)
             .build();
-    MockWebServer server;
-
-    @BeforeEach
-    void setup() {
-        server = new MockWebServer();
-    }
 
     @Test
     @DisplayName("Verify that expired EDR are deleted")
@@ -102,16 +92,6 @@ public abstract class AbstractDeleteEdrTest {
         await().atMost(ASYNC_TIMEOUT)
                 .untilAsserted(() -> expired.forEach((id) -> SOKRATES.edrs().getEdrRequest(id).statusCode(404)));
 
-    }
-
-    @AfterEach
-    void teardown() throws IOException {
-        server.shutdown();
-    }
-
-
-    private Condition<String> stateCondition(String value, String description) {
-        return new Condition<>(m -> m.equals(value), description);
     }
 
 }

--- a/gradle/libs.versions.toml
+++ b/gradle/libs.versions.toml
@@ -3,26 +3,27 @@ format.version = "1.1"
 
 [versions]
 edc = "0.5.1-20240131-SNAPSHOT"
-postgres = "42.7.1"
-awaitility = "4.2.0"
-nimbus = "9.37.3"
-azure-identity = "1.11.1"
-slf4j = "2.0.11"
-okhttp = "4.12.0"
-mockwebserver = "5.0.0-alpha.12"
-bouncyCastle-jdk18on = "1.77"
-restAssured = "5.4.0"
 apache-sshd = "2.12.0"
-testcontainers = "1.19.4"
-aws = "2.23.17"
-rsApi = "3.1.0"
-jupiter = "5.10.2"
 assertj = "3.25.3"
-titanium = "1.3.3"
+awaitility = "4.2.0"
+aws = "2.23.17"
+azure-identity = "1.11.1"
+bouncyCastle-jdk18on = "1.77"
+flyway = "9.22.3"
+iron-vc = "0.8.1"
 jackson = "2.16.1"
 jakarta-json = "2.0.1"
+jupiter = "5.10.2"
+mockwebserver = "5.0.0-alpha.12"
+nimbus = "9.37.3"
+okhttp = "4.12.0"
+postgres = "42.7.1"
+restAssured = "5.4.0"
+rsApi = "3.1.0"
+slf4j = "2.0.11"
+testcontainers = "1.19.4"
 tink = "1.12.0"
-iron-vc = "0.8.1"
+titanium = "1.3.3"
 
 [libraries]
 edc-spi-catalog = { module = "org.eclipse.edc:catalog-spi", version.ref = "edc" }
@@ -162,30 +163,31 @@ edc-transfer-receiver = { module = "org.eclipse.edc:transfer-pull-http-receiver"
 
 # other deps
 
-postgres = { module = "org.postgresql:postgresql", version.ref = "postgres" }
+apache-sshd-core = { module = "org.apache.sshd:sshd-core", version.ref = "apache-sshd" }
+apache-sshd-sftp = { module = "org.apache.sshd:sshd-sftp", version.ref = "apache-sshd" }
+apicatalog-iron-vc = { module = "com.apicatalog:iron-verifiable-credentials", version.ref = "iron-vc" }
+assertj = { module = "org.assertj:assertj-core", version.ref = "assertj" }
 awaitility = { module = "org.awaitility:awaitility", version.ref = "awaitility" }
-slf4j-api = { module = "org.slf4j:slf4j-api", version.ref = "slf4j" }
+aws-s3 = { module = "software.amazon.awssdk:s3", version.ref = "aws" }
+aws-s3transfer = { module = "software.amazon.awssdk:s3-transfer-manager", version.ref = "aws" }
+bouncyCastle-bcpkixJdk18on = { module = "org.bouncycastle:bcpkix-jdk18on", version.ref = "bouncyCastle-jdk18on" }
+flyway-core = { module = "org.flywaydb:flyway-core", version.ref = "flyway" }
+jackson-datatypeJsr310 = { module = "com.fasterxml.jackson.datatype:jackson-datatype-jsr310", version.ref = "jackson" }
+jacksonJsonP = { module = "com.fasterxml.jackson.datatype:jackson-datatype-jakarta-jsonp", version.ref = "jackson" }
+jakarta-rsApi = { module = "jakarta.ws.rs:jakarta.ws.rs-api", version.ref = "rsApi" }
+jakartaJson = { module = "org.glassfish:jakarta.json", version.ref = "jakarta-json" }
+junit-jupiter-api = { module = "org.junit.jupiter:junit-jupiter-api", version.ref = "jupiter" }
 nimbus-jwt = { module = "com.nimbusds:nimbus-jose-jwt", version.ref = "nimbus" }
 okhttp = { module = "com.squareup.okhttp3:okhttp", version.ref = "okhttp" }
 okhttp-mockwebserver = { module = "com.squareup.okhttp3:mockwebserver", version.ref = "mockwebserver" }
-bouncyCastle-bcpkixJdk18on = { module = "org.bouncycastle:bcpkix-jdk18on", version.ref = "bouncyCastle-jdk18on" }
+postgres = { module = "org.postgresql:postgresql", version.ref = "postgres" }
 restAssured = { module = "io.rest-assured:rest-assured", version.ref = "restAssured" }
-apache-sshd-core = { module = "org.apache.sshd:sshd-core", version.ref = "apache-sshd" }
-apache-sshd-sftp = { module = "org.apache.sshd:sshd-sftp", version.ref = "apache-sshd" }
+slf4j-api = { module = "org.slf4j:slf4j-api", version.ref = "slf4j" }
 testcontainers-junit = { module = "org.testcontainers:junit-jupiter", version.ref = "testcontainers" }
 testcontainers-postgres = { module = "org.testcontainers:postgresql", version.ref = "testcontainers" }
 testcontainers-vault = { module = "org.testcontainers:vault", version.ref = "testcontainers" }
-aws-s3 = { module = "software.amazon.awssdk:s3", version.ref = "aws" }
-aws-s3transfer = { module = "software.amazon.awssdk:s3-transfer-manager", version.ref = "aws" }
-jakarta-rsApi = { module = "jakarta.ws.rs:jakarta.ws.rs-api", version.ref = "rsApi" }
-jakartaJson = { module = "org.glassfish:jakarta.json", version.ref = "jakarta-json" }
-jacksonJsonP = { module = "com.fasterxml.jackson.datatype:jackson-datatype-jakarta-jsonp", version.ref = "jackson" }
-jackson-datatypeJsr310 = { module = "com.fasterxml.jackson.datatype:jackson-datatype-jsr310", version.ref = "jackson" }
-titaniumJsonLd = { module = "com.apicatalog:titanium-json-ld", version.ref = "titanium" }
-junit-jupiter-api = { module = "org.junit.jupiter:junit-jupiter-api", version.ref = "jupiter" }
-assertj = { module = "org.assertj:assertj-core", version.ref = "assertj" }
 tink = { module = "com.google.crypto.tink:tink", version.ref = "tink" }
-apicatalog-iron-vc = { module = "com.apicatalog:iron-verifiable-credentials", version.ref = "iron-vc" }
+titaniumJsonLd = { module = "com.apicatalog:titanium-json-ld", version.ref = "titanium" }
 
 [bundles]
 edc-connector = ["edc.boot", "edc.core-connector", "edc.core-controlplane", "edc.api-observability"]

